### PR TITLE
[DLSP21] 10-2

### PR DESCRIPTION
**Pull request by MrPhiDAP**
Week 10, Chapter 4 Barlow Twins

### CODE REVIEW

**Comment by Atcold** [comment]
I'm confused. What is this?

**Comment by MrPhiDAP** [comment]
Sorry professor for the confusion. I have changed the _config.yml to add the chapter barlow twins to week 10.

**Comment by MrPhiDAP** [comment]
https://mrphidap.github.io/NYU-DLSP21/en/week10/10-4/

File: docs/en/week10/10-4.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+lang-ref: ch.10-4
+title: Barlow Twins
+lecturer: Ishan Misra, Alfredo Canziani
```
Comment (Atcold) [inline docs/en/week10/10-4.md:4]:
```suggestion
lecturer: Ishan Misra
```

**Review by Atcold — CHANGES_REQUESTED** [review]
It looks very good, but I'm confused about what this is.
10–1 and 10–2 are part 1 and 2 of Ishan's lecture.
10-3 is my lab.
What is 10-4???

**Comment by MrPhiDAP** [comment]
@krishnakarthik9 i added your part to complete 10-2.md

**Comment by MrPhiDAP** [comment]
Alright @Atcold, i added @krishnakarthik9 to my pull request. This is the complete second part of week 10. Hence 10-2.md.

**Comment by MrPhiDAP** [comment]
https://mrphidap.github.io/NYU-DLSP21/en/week10/10-2/

File: docs/en/week10/10-2.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+<img src="{{site.baseurl}}/images/week10/10-2/cma.png" style="background-color:#DCDCDC;" /><br>
+Figure 4
+</center>
+
+## Distillation
+Distillation methods are similarity maximization based methods. Like other SSL methods distillation tries to prevent trivial solutions. It does so by asymmetry in two different ways.
+* Asymmetric *learning rule* between student teacher
+* Asymmetric *architecture* between student teacher
+
+$$ f_{\theta}^{student}(I) = f_{\theta}^{teacher}(augment(I))$$
```
Comment (Atcold) [inline docs/en/week10/10-2.md:45]:
```suggestion
$$ f_{\theta}^{\text{student}}(I) = f_{\theta}^{\text{teacher}}(\text{augment}(I))$$
```

File: docs/en/week10/10-2.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+### Implementation
+A successful approach to Self-Supervised-Learning (SSL) is to learn representations which are invariant to distortions of the input sample. However, a recurring problem with this approach is the existence of trivial constant solutions.
+
+The Barlow Twins method proposes an objective function that naturally avoids such collapse by measuring the cross-correlation matrix between the outputs of two identical networks fed with distorted versions of a sample and making them as close as possible to the identity matrix.
+
+Barlow's redundancy-reduction principle applied to a pair of identical networks. The objective function measures the cross-correlation matrix between the output features of two identical networks fed with distorted versions of a batch of samples and attemps to bring this matrix close to the identity. This causes the representation vectors of distorted versions of a sample to be similar, while minimizing the redundancy between the components of these vectors (Figure 7).
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-2/figure_1.png" style="background-color:#DCDCDC;" /><br>
+Figure 17 Barlow-Twins Architecture
```
Comment (Atcold) [inline docs/en/week10/10-2.md:80]: Maybe figure 7?

**Review by Atcold — CHANGES_REQUESTED** [review]
Very good. Figures do not have captions, though…
Also, feel free to invert also the first 6 ones. The 7th looks great!

**Comment by MrPhiDAP** [comment]
@krishnakarthik9 your part, please change the tables. If you need assistance, just contact me!

**Comment by krishnakarthik9** [comment]
Inverted images with lighter background and added captions. Changes are reflected here: https://mrphidap.github.io/NYU-DLSP21/en/week10/10-2/

**Comment by Atcold** [comment]
Very good, thanks!

**Pull request merged**